WORKSPACE

```diff
@@ -18,10 +18,9 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
     name = "cgrindel_bazel_starlib",
-    sha256 = "75ede4943a0000661cab0a4495bd10ddfc45bc73df83677eb75f49202ea09a34",
```

Comment:
im ok with dropping all the stuff in WORKSPACE and not running tests w/o bzlmod at this point, as long as we could verify using it at all w/o bzlmod still worked, maybe not worth the trouble to change it that way

Reply:
That works for me. I can put up another PR with a clean `WORKSPACE`. Right now, all of the examples use the legacy `WORKSPACE` loading. Do you want one that test bzlmod loading?